docs/src/content/docs/reference/std-widgets/overview.mdx

```diff
@@ -0,0 +1,118 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Overview
+description: Widgets Overview.
+---
+
+import CodeSnippetMD from '../../../../components/CodeSnippetMD.astro';
+import SlintProperty  from '../../../../components/SlintProperty.astro';
+
+```slint playground
+import { Palette, HorizontalBox } from "std-widgets.slint";
+
+export component MyCustomWidget {
+    in property <string> text <=> label.text;
+
+    Rectangle {
+        background: Palette.control-background;
+
+        HorizontalBox {
+            label := Text {
+                color: Palette.control-foreground;
+            }
+        }
+    }
+}
+```
+
+
+Slint provides a series of built-in widgets that can be imported from `"std-widgets.slint"`.
+
+The widget appearance depends on the selected style.
+See [Selecting a Widget Style](/master/docs/slint/reference/std-widgets/style) for details how to select the style. If no style is selected, `native` is the default. If `native` isn't available, `fluent` is the default.
```

Comment:
This is for solving later, but we somehow need to figure out what to do with these absolute links.

Is there any module in Astro that would allow for resolving these by name or some kind of relative reference?